common/flatpak-transaction.c

```diff
@@ -676,27 +681,38 @@ flatpak_transaction_operation_get_ref (FlatpakTransactionOperation *self)
 }
 
 /**
- * flatpak_transaction_operation_get_related_to_op:
+ * flatpak_transaction_operation_get_related_to_ops:
  * @self: a #FlatpakTransactionOperation
  *
- * Gets the operation which caused this operation to be added to the
- * transaction. In the case of a runtime, it's the app whose runtime it is. In
+ * Gets the operations which caused this operation to be added to the
+ * transaction. In the case of a runtime, it's the apps whose runtime it is (and
+ * this could be multiple apps, if they all require the same runtime). In
  * the case of a related ref such as an extension, it's the main app or
  * runtime. In the case of a main app or something added to the transaction by
- * flatpak_transaction_add_ref(), %NULL will be returned.
+ * flatpak_transaction_add_ref(), %NULL or an empty array will be returned.
```

Comment:
The way it's implemented we never return an empty array, so let's not say we do.

Reply:
Documenting it this way gives us the option to return an empty array in the future, if that would ease the implementation; but I can change it if you really want.

Reply:
Alright, sure